index.html

```diff
@@ -0,0 +1,484 @@
+            </div>
+            <div class="l-flex-item l-svcs">
+              <h3 class="f-bold l-flex-child">LABORATORY</h3>
+              <span class="icon-lab icon-big"></span>
+              <p class="l-flex-child-p">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </div>
+            <div class="l-flex-item l-svcs">
+              <h3 class="f-bold l-flex-child">DEVELOPMENT</h3>
+              <span class="icon-cog icon-big"></span>
+              <p class="l-flex-child-p">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </div>
+          </div>
+        </section>
+        <section>
+          <h3 class="f-bold h3-lines">Pricing Tables</h3>
+          <div class="l-flex-container">
+            <table class="l-flex-item l-flex-prices">
+              <thead>
+                <tr>
+                  <th class="f-light th-title">BASIC</th>
+                </tr></thead>
+              <tbody>
+                <tr>
+                  <td class="f-bold f-client">$5.99<span class="f-price"> / Month</span></td>
+                </tr>
+                <tr>
+                  <td>3 Projects</td>
+                </tr>
+                <tr>
+                  <td>2 Users</td>
+                </tr>
+                <tr>
+                  <td>1 Gb user</td>
+                </tr>
+                <tr>
+                  <td>1 Calendar</td>
+                </tr>
+              </tbody>
+              <tfoot>
+                <tr>
+                  <td class="td-btn"><a href=""><button class="btn f-bold">SIGN UP</button></a></td>
+                </tr>
+              </tfoot>
+            </table>
+            <table class="l-flex-item l-flex-prices">
+              <thead>
+                <tr>
+                  <th class="f-light th-title">STANDART</th>
+                </tr></thead>
+              <tbody>
+                <tr>
+                  <td class="f-bold f-client">$10.99<span class="f-price"> / Month</span></td>
+                </tr>
+                <tr>
+                  <td>7 Projects</td>
+                </tr>
+                <tr>
+                  <td>5 Users</td>
+                </tr>
+                <tr>
+                  <td>10 Gb user</td>
+                </tr>
+                <tr>
+                  <td>5 Calendar</td>
+                </tr>
+              </tbody>
+              <tfoot>
+                <tr>
+                  <td class="td-btn"><a href=""><button class="btn f-bold">SIGN UP</button></a></td>
+                </tr>
+              </tfoot>
+            </table>
+            <table class="l-flex-item l-flex-prices">
+              <thead>
+                <tr>
+                  <th class="f-light th-title">PROFESSIONAL</th>
```

Comment:
I won't comment on all of these (like I have with other items); just suffice it to say that I'd want all of the uses of `.th-title` to be taken in to consideration.